api/PclusterApiHandler.py

```diff
@@ -673,13 +674,30 @@ def login():
 
 
 def logout():
+    refresh_token = request.cookies.get('refreshToken', None)
+    if refresh_token is not None:
+        revoke_cognito_refresh_token(refresh_token)
+
     resp = __cognito_logout_redirect(get_app_config())
     resp.set_cookie("accessToken", "", expires=0)
     resp.set_cookie("idToken", "", expires=0)
     resp.set_cookie("refreshToken", "", expires=0)
     resp.set_cookie(CSRF_COOKIE_NAME, "", expires=0)
     return resp
 
+def revoke_cognito_refresh_token(refresh_token):
```

Comment:
issue: let's adapt the logout test to check this call is being performed

Reply:
Added in this new revision, also added tests for the function itself